vmdb/app/models/container_node.rb

```diff
@@ -8,4 +8,10 @@ class ContainerNode < ActiveRecord::Base
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
+
+  virtual_column :condition_ready, :type => :string, :uses => :container_node_conditions
+
+  def condition_ready
+    container_node_conditions.find_by_name('Ready').status
```

Comment:
@dkorn if container_node_conditions.find_by_name('Ready') is nil then doing.status on it will blow up, in that case you should return a string "None" to prevent from blowing up cc @abonas